Library/Homebrew/rubocops/platform.rb

```diff
@@ -0,0 +1,25 @@
+# typed: false
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Homebrew
+      # This cop ensures that platform specific code ends up in `extend/os`.
+      #
+      # @api private
+      class MoveToExtendOS < Base
+        MSG = "Move calls to `OS.linux?` and `OS.mac?` to `extend/os`."
```

Comment:
```suggestion
        MSG = "Move `OS.linux?` and `OS.mac?` calls to `extend/os`."
```

Sounds better this way. At least for me 😄

Reply:
Yeah, that does sound better. I'll wait until the end to change it just in case there are any other naming changes people want.

Reply:
Added in [aff4f27](https://github.com/Homebrew/brew/pull/14163/commits/aff4f271c39bac51a6b5822984766752bd1edece).